Marlin/src/lcd/language/language_uk.h

```diff
@@ -330,6 +361,14 @@ namespace Language_uk {
   PROGMEM Language_Str MSG_NOZZLE_STANDBY                  = _UxGT("Сопло очікує");
   PROGMEM Language_Str MSG_BED                             = _UxGT("Стіл,  ") LCD_STR_DEGREE "C";
   PROGMEM Language_Str MSG_CHAMBER                         = _UxGT("Камера,") LCD_STR_DEGREE "C";
+  PROGMEM Language_Str MSG_COOLER                          = _UxGT("Охолодження лазеру");
+  #if LCD_WIDTH > 21
```

Comment:
Of course, the width might only be 22. This value should probably be based on the actual length of the long string, and not just "21."